src/helm/tokenizers/cohere_tokenizer.py

```diff
@@ -9,26 +9,13 @@
     DecodeRequestResult,
     TokenizationToken,
 )
-from helm.clients.cohere_utils import get_cohere_url, DEFAULT_COHERE_API_VERSION
+from helm.clients.cohere_utils import get_cohere_url
 from .caching_tokenizer import CachingTokenizer
 
 
 class CohereTokenizer(CachingTokenizer):
-    # From "https://docs.cohere.ai/versioning-reference",
-    # "this version [2021-11-08] introduces multiple generations, meaning that the generations endpoint will
-    # now accept a num_generations argument in the JSON and will always return an array of generations"
-    # Note that the API version is decoupled from the model version.
-    DEFAULT_API_VERSION: str = "2021-11-08"
-
     TOKENIZE_ENDPOINT: str = "tokenize"
```

Comment:
You could consider using the Hugging Face tokenizer version of the Command R tokenizer to avoid using the API, [like so](https://github.com/stanford-crfm/helm/blob/79862464fb876408ad7e82fc8d7b45c5b2b76da0/src/helm/config/tokenizer_configs.yaml#L272-L279).

Reply:
Much cleaner, thank you!